core/src/main/java/com/linecorp/armeria/server/docs/DocServiceBuilder.java

```diff
@@ -168,10 +168,10 @@ private DocServiceBuilder exampleHttpHeaders0(String serviceName, String methodN
      * exampleRequest(serviceType.getName(), exampleRequests);
      * }</pre>
      */
-    public DocServiceBuilder exampleRequest(Class<?> serviceType, String methodName,
-                                            Object... exampleRequests) {
+    public DocServiceBuilder exampleRequestForMethod(Class<?> serviceType, String methodName,
+                                                     Object... exampleRequests) {
```

Comment:
I didn't remove any methods, just renamed them so they don't clash with `exampleRequest(Object...)`, which is still there to support Thrift.